wallet/wallet-rpc-daemon/docs/RPC.md

```diff
@@ -462,6 +462,16 @@ Returns:
              1) {
                     "num_valid_signatures": number,
                     "num_invalid_signatures": number,
+                    "signature_statuses": [ EITHER OF
+                         1) { "type": "NotSigned" }
+                         2) { "type": "InvalidSignature" }
+                         3) { "type": "UnknownSignature" }
+                         4) { "type": "FullySigned" }
+                         5) {
+                                "type": "PartialMultisig",
+                                "required_signatures": number,
+                                "num_signatures": number,
+                            }, .. ],
```

Comment:
Didn't we want to put these in a contents dict?

Reply:
updated